src/shared/views/BuildingFootprint/FootprintDisplay.js

```diff
@@ -56,7 +114,7 @@ export default class FootprintDisplay extends React.Component<Props> {
     };
 
     // return the center of the building footprint
-    getBuildingCentroid = (coords: Polygon): Point => {
+    getTaskGeometryCentroid = (coords: Polygon): Point => {
         const centroid: Point = coords
             .slice(0, -1)
             .reduce((acc, c) => [acc[0] + c[0], acc[1] + c[1]]);
```

Comment:
this is not needed for google imagery, but in general: what's your approach here?

"normally" you would derive the centroid of a polygon by computing it's bounding box and then you can get the center from these coordinates. So one need to find out max_lat, min_lat, max_lon, min_lon per geometry and from this get the centroid coordinates.

Reply:
what is `acc` and `c`?

Reply:
Ok, so my definition of centroid might be wrong then. I basically take the average of all latitude and same with longitudes, and that gives me a point which I guess is the centre of gravity of the geometry, probably not exactly the same as centroid. But I guess for this purpose, it's fine, it helps us put the shape roughly in the centre of the image.

Reply:
oh and `acc` is "accumulator" and `c` is just short for "coordinate" I think. See [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce) for more details.

Reply:
Actually, I'm wrong, it's not centre of gravity of the shape, more like the centre of gravity of the points...

Reply:
jo, I will create an issue for this okay? :) It's not urgent now and a good thing to fix for interested students of geography. ;)